linux_files/setup

```diff
@@ -891,6 +891,10 @@ if (whiptail --title "DOCKER" --yesno "Would you like to install the bridge to D
             else
                 sudo sed -i 's$\(root=\)\(.*\)$\1/$' /etc/wsl.conf
             fi
+
+	    # Temporary fix until WSLU fixes their path tool not updating when mount changed
+	    sudo rm -r /mnt/c
+	    sudo ln -s /c /mnt/c
```

Comment:
Ok,I tried a more permanent solution with mount --bind or modifying /etc/fstab but I couldn't find a good solution. Your solution is temporary as you said. I will look for a better solution soon.